projects/js-packages/publicize-components/src/components/services/use-supported-services.tsx

```diff
@@ -35,6 +42,11 @@ export function useSupportedServices(): Array< SupportedService > {
 			);
 	}, [] );
 
+	const badgeNew: Badge = {
+		text: __( 'New', 'jetpack' ),
+		style: { background: '#e9eff5', color: '#0675C4' },
```

Comment:
It might be best to wait for the design, but do you think we should use some already defined Jetpack colours here?

Reply:
I used the colors mentioned in the designs - p1HpG7-swB-p2

But those colors are not defined anywhere in Jetpack.